setup.py

```diff
@@ -19,4 +21,76 @@
           'ed25519',
           'python-bitcoinlib'
       ],
-      zip_safe=False)
+      zip_safe=False,
+      long_description="""\
```

Comment:
For sake of organization (and easier editing when making changes) we should move all this description text into a separate file like `pypi_long_description.md` or whatever other filetype it is and reference it like this:
```python
from setuptools import setup
import os

def long_description_file():
    with open('pypi_long_description.md') as f:
        return f.read()

setup(name='python-identitykeys',
...
      long_description=long_description_file()
)
```
https://python-packaging.readthedocs.io/en/latest/metadata.html